### ISSUE TITLE
Missing formatting on links in admonitions

### ISSUE THREAD

**Issue by janosdebugs** [opentofu/opentofu.org#223]
When adding links to an admonition no styling is present, making links indistinguishable from normal text.

**Comment by IgnorantSapient** [comment]
Is this still an issue? Has this been fixed? I feel the links are distinguishable in ........

<img width="710" alt="image" src="https://github.com/opentofu/opentofu.org/assets/4549937/2feac54f-04ad-43b3-87df-6f80aeed373f">

This is probably a stale issue and it needs to be closed

**Comment by janosdebugs** [comment]
Hi @IgnorantSapient that's not an adminition. Admonitions are the little info/warning boxes. As far as I known this isn't fixed.

**Comment by janosdebugs** [comment]
Unfortunately, this issue is still present:

![image](https://github.com/opentofu/opentofu.org/assets/86970079/9241a15c-db32-4c15-bbba-0dd34d4fff1e)

Repro code:

```
:::note

Here's some text [here's a link](https://example.com)

:::
```

**Comment by IgnorantSapient** [comment]
@janosdebugs Thanks so much for the clear explanation, A fix has been initiated here: #258
